lib/api/objectPutCopyPart.js

```diff
@@ -252,6 +258,30 @@ function objectPutCopyPart(authInfo, request, sourceBucket,
                         stream.pipe(hashedStream);
                         const numberPartSize =
                             Number.parseInt(part.size, 10);
+
+                        if (externalBackends[config.locationConstraints
```

Comment:
please add comment as to why we are only using objectLocationConstraint here instead of controlling location constraint